QtlDataPass/ContentView.swift

```diff
@@ -8,12 +8,29 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State var textSecond = ""
+    @State var textThird = ""
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            NavigationStack {
+                Spacer()
+                TextField("文字を入力", text: $textSecond)
+                    .textFieldStyle(RoundedBorderTextFieldStyle())
+                NavigationLink(destination: SecondView(textSecond: $textSecond)) {
+                    Text("SecondViewに画面遷移")
+                }.padding()
```

Comment:
```suggestion
                NavigationLink {
                    SecondView(textSecond: $textSecond)
                } label: {
                    Text("SecondViewに画面遷移")
                }
                .padding()
```

[imo] こちらの方が、読みやすくなります
- Swiftlintだと元の書き方でwarningかerrorが出たような...?

Reply:
コミットハッシュ: c2e095ab8f744f2d57332797fb7c2515a148b25e

変更: TopViewの26行目 ~ 30行目で修正